src/Annotation/AnnotationCollection.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+namespace Laminas\Form\Annotation;
+
+use ArrayObject;
+
+use function get_class;
+
+/**
+ * @extends ArrayObject<array-key, object>
+ */
+final class AnnotationCollection extends ArrayObject
+{
+    /**
+     * Checks if the collection has annotations for a class
+     *
+     * @param  string $class
+     * @return bool
+     */
+    public function hasAnnotation($class)
```

Comment:
Type declarations and return types should be specified for all new methods.

Reply:
@froschdesign should we, given that this goes in 3.0 anyways, add return types and type declarations to all methods?

Reply:
Indeed, that's what I want too: https://github.com/laminas/laminas-form/pull/111#discussion_r637832963

Reply:
If the methods are new, why wait?

Reply:
Done